src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../../common/variables';
+
+.bcs-AnnotationActivityLink {
+    display: inline-block;
+    overflow-x: hidden;
```

Comment:
Do we need this on both the parent and the message?

Reply:
Yeah seems like it